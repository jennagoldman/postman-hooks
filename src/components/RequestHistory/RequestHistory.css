h2 {
  color: #fc40e6;
}

.requestMethod {
  text-transform: uppercase;
}