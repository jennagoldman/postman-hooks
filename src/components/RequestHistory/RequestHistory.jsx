import React from 'react';
import styles from './RequestHistory.css';

const RequestHistory = () => (
  <div className={styles.historyContainer}>
    <h2>Request History</h2>
  </div>
);

export default RequestHistory;
