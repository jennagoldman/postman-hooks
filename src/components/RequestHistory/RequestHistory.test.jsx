import React from 'react';
import { shallow } from 'enzyme';
import RequestHistory from './RequestHistory.jsx';

describe('RequestHistory', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<RequestHistory history={[]} onReset={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
