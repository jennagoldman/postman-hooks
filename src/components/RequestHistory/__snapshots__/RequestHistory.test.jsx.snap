// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RequestHistory matches a snapshot 1`] = `
<section
  className="historyContainer"
>
  <h2>
    Request History
  </h2>
  <button
    onClick={[Function]}
  >
    Clear
  </button>
  <ul />
</section>
`;
